app-vim/brainfuck-syntax/brainfuck-syntax-1.0.3-r2.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
```

Comment:
I'd say these trivial EAPI bumps should be left to vim@ to handle (i.e. drop this commit). Back in April these received a mass bump from 5 -> 7 and I imagine 6 -> 7 (or even 8 if support is added) may happen eventually.

Also, for a trivial vim ebuild, I'd say revbump for a simple EAPI bump with no other changes accomplishes nothing but give more work to arch testers.

Reply:
If you take a look on Manifest is not only EAPI update but also version update (from pre20180125 to pre20210215). If you prefer I can drop the -1.0.3-r2 and use release date